ci/edmtool.py

```diff
@@ -105,6 +105,10 @@
     "unittest2",
     "pypdf2",
     "swig",
+    # required by some demos
+    "chaco",
+    "mayavi",
+    "scipy",
```

Comment:
I didn't make this optional because the `install` command sets up the dev environment which probably should have all dependencies.